discord-scripts/invite-management.ts

```diff
@@ -109,71 +69,75 @@ export default async function sendInvite(discordClient: Client, robot: Robot) {
           })
           await interaction.reply(
             `Here is your invite link: ${invite.url}\nThis invite expires in ${
-              maxAge / 3600
-            } hours and has a maximum of ${maxUses} uses.`,
+              maxAge / (3600 * 24)
```

Comment:
One of the lovely properties of our constants is this is synonymous with `maxAge / DAYS`, i.e. they can be used both as a multiplier and a divisor.